client/Android/Studio/freeRDPCore/src/main/res/values-de/strings.xml

```diff
@@ -80,9 +80,9 @@
         <item>1920x1200</item>
     </string-array>
     <string-array name="resolutions_values_array">
-        <item>automatisch</item>
-        <item>an Bildschirm anpassen</item>
-        <item>Benutzerdefiniert</item>
+        <item>automatic</item>
+        <item>fitscreen</item>
+        <item>custom</item>
```

Comment:
@bmiklautz But you did give me an idea, looks like non translatable strings can be marked ;)